src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/CallbackParams.java

```diff
@@ -14,6 +14,6 @@
     private Map<Params, Object> params;
 
     public enum Params {
-        BEARER_TOKEN
+        BEARER_TOKEN, SERVICE_AUTH_TOKEN
```

Comment:
Can't find any usage for `SERVICE_AUTH_TOKEN`

Reply:
ah yes, sorry, I had added it to verify it was sent in the callback for securing the endpoint :D I'll remove it